templates/curl/template.json

```diff
@@ -32,7 +32,6 @@
                         "Resources": {
                             "CPU": 50,
                             "MemoryMB": 128,
-                            "DiskMB": 500,
```

Comment:
Why are you removing this?

Reply:
Cause Nomad wont deploy the job with this parameter.

Reply:
@FRosner 
Cause Nomad wont run the job with this line as it was.
Here is Nomad output:
```bash
    default: [NOMAD]     2018/03/16 14:23:56.358846 [ERR] http: Request /v1/jobs, error: 1 error(s) occurred:
    default: [NOMAD]
    default: [NOMAD] * Task group curl-group validation failed: 1 error(s) occurred:
    default: [NOMAD]
    default: [NOMAD] * Task curl-task validation failed: 1 error(s) occurred:
    default: [NOMAD]
    default: [NOMAD] * Task can't ask for disk resources, they have to be specified at the task group level.

```

Reply:
Ah so this is actually something that changed going to a newer Nomad version it seems. I remember that they require you to specify disk info in a different location from now on.